tests/end2end/features/navigate.feature

```diff
@@ -18,6 +18,22 @@ Feature: Using :navigate
         And I run :navigate up with count 2
         Then data/navigate should be loaded
 
+    Scenario: Navigating up in qute://help/
+        When I open qute://help/commands.html
+        And I run :navigate up
+        Then qute://help/ should be loaded
+
+    Scenario: Navigating up by count in qute://help/
```

Comment:
I'm not sure this test is useful - do you think it tests something the normal "Navigating up by count" test does not?

Reply:
I don't know really, I added this for completeness' sake. I'll remove it.